tap-setup-scripts/src/cloud-init.sh

```diff
@@ -62,8 +62,8 @@ systemctl start amazon-cloudwatch-agent.service
 systemctl status amazon-cloudwatch-agent.service
 echo "Installing kubectl..."
 pushd /tmp
-aws s3 cp --no-progress "s3://amazon-eks/${AwsKubectlVersion}/bin/linux/$arch/kubectl" ./kubectl
-aws s3 cp --no-progress "s3://amazon-eks/${AwsKubectlVersion}/bin/linux/$arch/kubectl.sha256" ./kubectl.sha256
+curl --retry 5 -o ./kubectl "https://s3.us-west-2.amazonaws.com/amazon-eks/${AwsKubectlVersion}/bin/linux/$arch/kubectl"
+curl --retry 5 -o ./kubectl.sha256 "https://s3.us-west-2.amazonaws.com/amazon-eks/${AwsKubectlVersion}/bin/linux/$arch/kubectl.sha256"
 openssl sha1 -sha256 ./kubectl
```

Comment:
*question*: what's the use of pulling down the checksum file, if we don't validate our artifact?

Instead of the `openssl` call (not sure what its purpose is), do we want to do something like `sha256 --check kubectl.sha256`?